Package coq-matrix.1.0.1

zhengpushi opened the pull request:
Sorry, coq-matrix.1.0.0 has a build problem. Now, I have fixed this problem and want to publish coq-matrix.1.0.1 to the community.
Change Log
1. remove the dependency for CoqExt library, which is charged by myself.
2. add dependency of coq-mathcomp-algebra, >=8.15.0

palmskog commented:
Hm, I guess I missed the continuous integration report. @zhengpushi, can you also please remove the 1.0.0 package then, since it doesn't build?

zhengpushi commented:
> Hm, I guess I missed the continuous integration report. @zhengpushi, can you also please remove the 1.0.0 package then, since it doesn't build?

Thank you. I have deleted this package.

silene commented:
> I guess I missed the continuous integration report.

No idea why, but the previous pull request did not trigger an actual build. This time, it seems to work.

zhengpushi commented:
I have noticed the fail report. Test has failed only at opam-build:4.07.1. I will check the error and fix it quickly.

palmskog commented on released/packages/coq-matrix/coq-matrix.1.0.1/opam:
```diff
@@ -28,6 +28,7 @@ license: "MIT" # Make sure this is reflected by a LICENSE file in your sources
 
 depends: [
   "coq" {>= "8.13"}
```
```suggestion
  "coq" {>= "8.16"}
```

palmskog reviewed (commented):
@zhengpushi in the continuous integration, it looks like the package is simply not compatible with Coq 8.15 (which is the version used for OCaml 4.07.1). How about we simply increase the lower bound for Coq to 8.16?

zhengpushi commented:
> @zhengpushi in the continuous integration, it looks like the package is simply not compatible with Coq 8.15 (which is the version used for OCaml 4.07.1). How about we simply increase the lower bound for Coq to 8.16?

Thank you for your help. But I have reproduced this bug under OCaml 4.07.1 environment, and changed my code finally. 
Now I want to pull another new version coq-matrix.1.0.2, and I have removed these old version.
